src/components/bootstrap/bootstrap.scss

```diff
@@ -0,0 +1,5 @@
+@import url('~bootstrap/dist/css/bootstrap.css');
+
+.container-fluid {
+  padding: 0;
```

Comment:
Why did you need to overwrite bootstraps default style here?

Edit: since you're using `.container-fluid`, you can use `.row` instead of `.row-fluid`. `.row` contains a negative margin that will offset `.container-fluid`'s extra padding, so you don't need to overwrite anything here.